ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml

```diff
@@ -231,3 +231,14 @@ ethernet_interfaces:
     mode: trunk
     vlans: 110-112
     trunk_private_vlan_secondary: true
+
+  Ethernet18:
+    description: 200MBit/s shape
+    shape:
+      rate: 200000
+
+  Ethernet19:
+    description: 10% shape
+    shape:
+      rate: "10"
```

Comment:
Please keep it integer although type conversion is possible. Also remove extra new line at the end of this file